src/components/frame/v5/pages/UserCryptoToFiatPage/partials/LiquidationAddress/LiquidationAddress.tsx

```diff
@@ -0,0 +1,158 @@
+import { Cardholder, Eye } from '@phosphor-icons/react';
+import clsx from 'clsx';
+import React, { useMemo, useState } from 'react';
+import { defineMessages } from 'react-intl';
+
+import LoadingSkeleton from '~common/LoadingSkeleton/index.ts';
+import { useAppContext } from '~context/AppContext/AppContext.ts';
+import { useGetUserLiquidationAddressesQuery } from '~gql';
+import useCopyToClipboard from '~hooks/useCopyToClipboard.ts';
+import { getChainId } from '~redux/sagas/utils/index.ts';
+import { formatText } from '~utils/intl.ts';
+import Button from '~v5/shared/Button/index.ts';
+
+import { useCryptoToFiatContext } from '../../context/CryptoToFiatContext.ts';
+
+import { MOCK_LIQUIDATION_ADDRESSES } from './mocks.ts';
+import { BlockExplorerButton } from './partials/BlockExplorerButton.tsx';
+
+const displayName = 'v5.pages.UserCryptoToFiatPage.partials.LiquidationAddress';
+
+const chainId = getChainId();
```

Comment:
I hope this is the right source to get the Chain ID. I've seen another way for grabbing the chain ID in `src/redux/sagas/users/index.ts`:
```js
const { chainId } = yield colonyManager.provider.getNetwork();
```
I compared the 2 values and they're the same, only the type is different, string versus number